src/main/java/organice/logic/commands/DoneCommand.java

```diff
@@ -0,0 +1,107 @@
+package organice.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static organice.logic.commands.MatchCommand.match;
+
+import organice.model.Model;
+import organice.model.person.Donor;
+import organice.model.person.Nric;
+import organice.model.person.Patient;
+import organice.model.person.exceptions.PersonNotFoundException;
+
+/**
+ * Set the status of a donor and patient to Done and remove the pair from the model
+ */
+public class DoneCommand extends Command {
+    public static final String COMMAND_WORD = "done";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finish the processing of patients and donors"
+            + "Parameters: ic/PATIENT NRIC ic/DONOR NRIC \n"
+            + "Example: " + COMMAND_WORD + " ic/s4512345A ic/s7711123C";
+
+    public static final String MESSAGE_SUCCESS = "Done processing the patient and donor";
+    public static final String MESSAGE_NOT_PROCESSED = "Donor or patient NRIC must be valid";
+
+    private String firstNricString;
+    private String secondNricString;
```

Comment:
Yo this one dont need save, can just use `nric.toString()`

Reply:
Oh but this one is like the "user input"?

Reply:
Adding on to William's point, in the constructor we made two `Nric` objects with the string inputs and saved it in the class as instance variables. 
If you want to access the String value of the nric objects, you can just use `nric.toString()`. So there is no need to create String variables of the nric.

Reply:
I mean the string variable are from the user input. because the method take in 2 nric in strings. then from there, the method change it to Nric then patientnric and donornric respectively.